app/components/article-featured-video.js

```diff
@@ -4,11 +4,13 @@ import duration from '../helpers/duration';
 import {track, trackActions} from '../utils/track';
 import extend from '../utils/extend';
 
-const {Component, inject} = Ember;
+const {Component, inject, run} = Ember;
 
 export default Component.extend(
 	{
 		classNames: ['article-featured-video'],
+		classNameBindings: ['isPlayerLoading::player-ready', 'isPlayed:is-played', 'isVideoDrawerVisible:fixed'],
```

Comment:
if isPlayed is bool you can ommit is-played
https://ember-twiddle.com/b5722f90744715f88ae59d2ea92ab5e3?openFiles=components.my-component.js%2Ctemplates.components.my-component.hbs

it's standard in Ember